### ISSUE TITLE
not working anymore in 2121

### ISSUE THREAD

**Issue by mouton-rebelle** [atombender/sublime_text_alternative_autocompletion#4]
I guess it's the cons of working with work in progress / beta software... I love your plugin, any chance you can fix it ?

**Comment by atombender** [comment]
The current build is 2111. Where did you get 2121?

**Comment by mouton-rebelle** [comment]
auto-upgrade on the purchased version... 
http://www.sublimetext.com/dev

**Comment by atombender** [comment]
Ok. I prefer to use the non-dev builds myself. If you can open the Sublime Text console and give me the error (including stack trace) I can look into the problem.

**Comment by mouton-rebelle** [comment]
I just found the issue ; num_selections is broken, so the keybinding is broken too
will stick to the beta version for the next time :)

thx

**Comment by atombender** [comment]
All right. Thanks.

**Comment by ehamiter** [comment]
Hi Alex,

I noticed that if I took out these sections in my keybinding:

```
 { "key": "num_selections", "operator": "equal", "operand": 1 },
```

then it works properly. (Discovered here: http://www.sublimetext.com/forum/viewtopic.php?f=2&t=3192#p15092)

**Comment by ehamiter** [comment]
...and fixed in #2122.

**Comment by atombender** [comment]
Cool!
